common.go

```diff
@@ -98,3 +102,21 @@ func (a ACL) And(session ...proto.SessionType) ACL {
 func (t ACL) Includes(session proto.SessionType) bool {
 	return t&ACL(1<<session) != 0
 }
+
+func VerifyACL[T any](acl Config[ACL]) error {
```

Comment:
What is purpose of this function? Let's not use reflect please.

Reply:
Verify that your access control map is filled, it is used for testing